samples/security/spire/sleep-spire.yaml

```diff
@@ -47,6 +47,7 @@ spec:
     metadata:
       labels:
         app: sleep
+        spiffe.io/spiffe-id: "true"
```

Comment:
See above - would prefer `spiffe.io/spire-managed-identity: "true"` to make it clearer this is specifically saying that SPIRE is attesting and managing the workload identity.

Reply:
Updated!